Tests/Form/DataTransformer/ModelToIdPropertyTransformerTest.php

```diff
@@ -195,7 +195,7 @@ public function testTransformMultiple()
         $collection[] = $entity2;
         $collection[] = $entity3;
 
-        $this->modelManager->expects($this->exactly(3))
+        $this->modelManager->expects($this->exactly(6))
```

Comment:
The number of calls to the mocked method has changed from 3 calls to 6 calls.

The supplementary calls are added by the last line which calls `transform` again.